form/remote-validation/readme.md

```diff
@@ -0,0 +1,28 @@
+# Remote (server, async) Validation upon Input in Form
+
+The samples in this folder showcase how you can use remote validation on the server to prevent inputs when they don't satisfy certain conditions that cannot be implemented through the standard `DataAnnotation` validation.
```

Comment:
Prefer shorter and simpler sentences:

```suggestion
The samples in this folder showcase how to use remote validation on the server to prevent invalid user input. This is useful when validation conditions cannot be implemented through standard `DataAnnotation` validation.
```